migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -73,6 +73,82 @@ For more information, see xref:../updating/updating-cluster-between-minor.adoc#u
 
 Review the changes and other considerations that might affect your transition from {product-title} 3.11 to {product-title} 4.
 
+[id="migration-DNS-considerations"]
```

Comment:
This content does not belong in this assembly. The assembly is about differences between OCP 3 and 4. The part that you are adding is about how to solve the problem of the clusters having different DNS domains. It needs to be put into a module, not this assembly.